src/sentry/integrations/utils/code_mapping.py

```diff
@@ -285,6 +283,27 @@ def _potential_match(self, src_file: str, frame_filename: FrameFilename) -> bool
             match = (
                 split[0].rfind(f"/{frame_filename.root}") > -1 or split[0] == frame_filename.root
```

Comment:
IIUC, this is the main difference in the transformation - we might be able to clean up the way this is split at a later point.